### PR TITLE
feat(seminar): add draft page of seminar summer 2024

### PR DESCRIPTION
**Pull request by rockleona**
<details>
<summary>Preview</summary>

image

</details>

### CODE REVIEW

File: content/pages/conference/2024/06-seminar.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+* Date: 22nd June, Saturday, 2024
+* Time: 8:30 -- 17:40
+
+|
+
+Talks in seminar
+----------------
+
+* Visualization that balances performance and flexibility for scientific computing by **Chun-Hsu Lai**
```
Comment (yungyuc) [inline content/pages/conference/2024/06-seminar.rst:20]: Confirmed topics should be made explicit and separate from unconfirmed ones.  Check the workbook.

File: content/pages/conference/2024/06-seminar.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* How does sciwork implement web development in CI/CD way? by **Chester**
+* Cytnx Development Update by **高英哲** and **陳柏中**
```
Comment (yungyuc) [inline content/pages/conference/2024/06-seminar.rst:22]: The topic of Cytnx is confirmed.  Please take a look at the workbook.

File: content/pages/conference/2024/06-seminar.rst [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+|
+
+Talks in seminar
+----------------
+
+* Visualization that balances performance and flexibility for scientific computing by **Chun-Hsu Lai**
+* How does sciwork implement web development in CI/CD way? by **Chester**
+* Cytnx Development Update by **高英哲** and **陳柏中** 
+* Python Documentation Translation by **Matt Wang** and **Li-Hung Wang**
```
Comment (yungyuc) [inline content/pages/conference/2024/06-seminar.rst:23]: Please list only speaker names for the talk whose topics are not confirmed.

**Review by yungyuc — CHANGES_REQUESTED** [review]
- [x] Make confirmed topics explicit.
- [x] Update topics.
- [ ] For talks without confirmed topics, list only speaker names.
- [ ] Remove unconfirmed talks.

Please make this PR ready for review since you are requesting review.  Make a global comment when you update the contents and ready for the follow-up review.

**Comment by rockleona** [comment]
> * [ ]  Make confirmed topics explicit.
> * [ ]  Update topics.
> * [ ]  For talks without confirmed topics, list only speaker names.
> * [ ]  Remove unconfirmed talks.
> 
> Please make this PR ready for review since you are requesting review. Make a global comment when you update the contents and ready for the follow-up review.

I've confirmed that these items are already updated, please review.

**Comment by yungyuc** [comment]
LGTM.  Merge.

**Pull request merged**